spec/controllers/api/v1/sentences_controller_spec.rb

```diff
@@ -8,7 +8,7 @@
   context 'on GET to index' do
     context 'requesting json representation', api_specification: true do
       let(:sentences_schema) do
-        schema_for('ontology/commands/sentences')
+        schema_for_command('ontology/sentences', :get, 200)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.